src/contexts/SDKContext.ts

```diff
@@ -51,6 +52,7 @@ export class SdkContextClass {
     public client?: MatrixClient;
 
     // All protected fields to make it easier to derive test stores
+    protected _WidgetPermissionStore?: WidgetPermissionStore;
```

Comment:
Does not matter for this PR:

We should not use the `_` style an more. In this particular place we could use something like `cachedWidgetPermissionStore`. Furthermore I found the test class. Instead of tricking with the visibilities we could instead provide a function that creates something that has the same public interface for tests.

Reply:
I used _ simply for brevity when writing code, given i cannot use lowerCaseStore as the getter function has that name. Happy with whatever name we want given it isn't really exposed to the rest of the app, though I'd personally prefer it if they were short.

I like switching the visibilities over because it means tests will exercise the same code in `SdkContextClass` as production. I don't see the value in interfacing this personally, I just see this as writing unnecessary boilerplate?